mentirinha.env

```diff
@@ -10,3 +10,7 @@ DJANGO_ALLOWED_HOSTS=localhost
 DJANGO_DEBUG=1
 DJANGO_SECRET_KEY=secret_key
 DJANGO_LOG_FILE=/dkdata/mentirinha.log
+
+# API Authentication
+# Set this to a secure random token for the /api/shorten endpoint
+API_TOKEN=your-secret-api-token-here
```

Comment:
Why exactly do we need authentication for this internal tool? A possible option (with less code) is to block external requests for this specific endpoint from within the firewall.

However, doing that separates a specific business logic (if it is) from the code itself.

I trust your judgement on this, though.